### PR TITLE
Split Parking Ticket type into Parking Ticket and Civil Offense

### PR DESCRIPTION
**Pull request by wittejm**
- A charge is now only classified as a ParkingTicket if the Case `violation_type` contains 'parking'
 - Chapters 1-99 now become classified instead as CivilOffense
 - Civil Offenses that are also Violations are classified as Violations with higher priority, because Violations have eligibility under 5d, whereas CivilOffenses are never eligible.

### CODE REVIEW

**Comment by wittejm** [comment]
Closes #764

File: src/backend/expungeservice/models/charge_types/parking_ticket.py [hunk 1]
```diff
@@ -6,10 +6,16 @@
...
     type_name: str = "Parking Ticket"
 
     def _default_type_eligibility(self):
-        return TypeEligibility(EligibilityStatus.INELIGIBLE, reason="Ineligible under 137.225(5)")
-
+        if self.convicted():
+            return TypeEligibility(EligibilityStatus.INELIGIBLE, reason="Ineligible under 137.225(7)(a)")
+        elif self.acquitted():
+            return TypeEligibility(EligibilityStatus.INELIGIBLE, reason="Ineligible by omission from statute")
     def skip_analysis(self):
```
Comment (KentShikama) [inline src/backend/expungeservice/models/charge_types/parking_ticket.py]: Preferred space before method

File: src/backend/tests/models/charge_types/test_civil_offense.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+class TestCivilOffense(unittest.TestCase):
+    def setUp(self):
+        last_week = datetime.today() - timedelta(days=7)
+        self.convicted = ["Convicted", last_week]
+        self.dismissed = ["Dismissed", last_week]
+
+    def test_00_is_not_a_civil_offense(self):
+        charge = ChargeFactory.create(statute="00", level="N/A", disposition=self.convicted)
+
+        assert charge.__class__.__name__ != "CivilOffense"
```
Comment (KentShikama) [inline src/backend/tests/models/charge_types/test_civil_offense.py]: Use `instanceof`

File: src/backend/tests/models/charge_types/test_parking_ticket.py [hunk 3]
```diff
@@ -1,32 +1,39 @@
...
-        self.build()
 
-        assert self.parking_ticket.skip_analysis() is True
+    def test_parking_ticket_dismissal(self):
+        self.charge_dict["disposition"]=self.dismissed
+        charge = ChargeFactory.create(**self.charge_dict)
 
-    def test_acuitted_tickets_are_not_type_eligible(self):
-        self.build(ruling="Dismissed")
+        assert charge.__class__.__name__ == "ParkingTicket"
```
Comment (KentShikama) [inline src/backend/tests/models/charge_types/test_parking_ticket.py]: Same here

File: src/backend/tests/models/charge_types/test_parking_ticket.py [hunk 4]
```diff
@@ -1,32 +1,39 @@
...
-        self.build()
+    def test_parking_ticket_conviction(self):
+        self.charge_dict["disposition"]=self.convicted
+        charge = ChargeFactory.create(**self.charge_dict)
 
-        assert self.parking_ticket.expungement_result.type_eligibility.status is EligibilityStatus.INELIGIBLE
-        assert self.parking_ticket.expungement_result.type_eligibility.reason == "Ineligible under 137.225(5)"
-        assert self.parking_ticket.type_name == "Parking Ticket"
+        assert charge.__class__.__name__ == "ParkingTicket"
+        assert charge.skip_analysis() is True
```
Comment (KentShikama) [inline src/backend/tests/models/charge_types/test_parking_ticket.py]: `assert charge.skip_analysis()`

**Pull request merged**